lib/fetching/content-services/crowd-tangle-content-service.js

```diff
@@ -79,7 +79,8 @@ CrowdTangleContentService.prototype._httpRequest = function(params, callback) {
 
 CrowdTangleContentService.prototype._completeUrl = function() {
 	// add one second to last report date to start fetching from
-	var startDate = new Date(this._lastReportDate.getTime() + 1000).toISOString();
+	// if it is the first time fetching data, initialize start date to an old date
+	var startDate = this._lastReportDate ? new Date(this._lastReportDate.getTime() + 1000).toISOString() : new Date("January 1, 1995");
```

Comment:
I see what you did here; looks great. However, I think we might need the logic to be different. If lastReportDate is null (meaning first fetch), we should not go very far back in time (as you have here), else we may never get to the most recent content. I'd rather propose that we set lastReportDate to be => now - someShortIntervalInHours (where 1 <= someShortIntervalInHours <= 6). I'm not sure the best way to choose that number but I will stick with 2-3 and change accordingly when we study traffic.

Reply:
Thanks for the input Daniel, that makes sense. I have made the changes you have requested. Could you check the [change](https://github.com/TID-Lab/aggie/pull/336/commits/d7a33381dd95ff64e5701a56b00914532ff20c0d)?